lib/Stripe/BitcoinTransaction.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+class Stripe_BitcoinTransaction extends Stripe_ApiResource
```

Comment:
Is the intent here that bitcoin transactions will properly have their own object, but you won't be able to do anything special with them like save?